.travis.yml

```diff
@@ -1,6 +1,23 @@
-language: node_js
-node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-  - "4.2.4"
+language: c++
+env:
+  - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
+  - NODE_VERSION="0.12"
+  - NODE_VERSION="4.2.4"
+  - NODE_VERSION="5.0"
```

Comment:
Ok, I can switch to 5. I wasn't sure if we always wanted to test against the latest release of node (could be somewhat unstable).